notebooks/basics/single-systems.ipynb

```diff
@@ -645,15 +646,15 @@
     "is denoted and defined as follows:\n",
     "\n",
     "$$\n",
-    "  \\| v \\| = \\sqrt{\\sum_{k=1}^n |\\alpha_k|^2}.\n",
+    "  \\| v \\| := \\sqrt{\\sum_{k=1}^n |\\alpha_k|^2}.\n",
```

Comment:
I prefer not to use the symbol :=. I have mixed feelings about it, and just haven't been able to figure out how to use it consistently.